library/kotlin/src/io/envoyproxy/envoymobile/RequestTrailers.kt

```diff
@@ -0,0 +1,14 @@
+package io.envoyproxy.envoymobile
+
+/*
+ * Trailers representing an outbound request.
+ */
+@Suppress("EmptyClassBlock")
```

Comment:
Is this actually needed? We can add a TODO for removing this lint check

Reply:
same as other comment